### ISSUE TITLE
[feat]: Request to Add Locale-Specific Date-Time Input Support in shadcn/ui Components. 

### ISSUE THREAD

**Issue by M-Ahmed-dev** [shadcn-ui/ui#5224]
### Feature description

You guys have created whole library but there's no component that can replace the need to use native input type = "datetime-locale".  There should be a similar component that let the user easily enter both a date and a time, including the year, month, and day as well as the time in hours and minutes.

### Additional Context

Additional details here...

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues and PRs

**Comment by huybuidac** [comment]
@M-Ahmed-dev You can try this https://github.com/huybuidac/shadcn-datetime-picker

demo: https://shadcn-datetime-picker-xi.vercel.app

![image](https://github.com/user-attachments/assets/dd39b448-5860-4a5c-96db-59dcade43684)

**Comment by M-Ahmed-dev** [comment]
Thank you man. Really appreciate it . 😊

**Comment by carlos-lopez-SaRa** [comment]
@huybuidac Have you seen any example of one like MUI's Date picker that allows the user to type out the whole date? https://mui.com/x/react-date-pickers/date-picker/

**Comment by huybuidac** [comment]
> @huybuidac Have you seen any example of one like MUI's Date picker that allows the user to type out the whole date? https://mui.com/x/react-date-pickers/date-picker/

Yep, this is the next feature I’m tackling. It’s a bit tricky, but fun times!

**Comment by huybuidac** [comment]
> @huybuidac Have you seen any example of one like MUI's Date picker that allows the user to type out the whole date? https://mui.com/x/react-date-pickers/date-picker/

let try http://localhost:6006/?path=/story/datetime-picker--date-time-input-picker

**Comment by carlos-lopez-SaRa** [comment]
Uh not sure what you're trying to link with a localhost url...

**Comment by huybuidac** [comment]
> Uh not sure what you're trying to link with a localhost url...

updated bro

**Comment by shadcn** [comment]
This issue has been automatically marked as stale due to two years of inactivity. It will be closed in 7 days unless there’s further input. If you believe this issue is still relevant, please leave a comment or provide updated details. Thank you.

**Comment by shadcn** [comment]
No stale. Something went wrong with the stale bot.

**Comment by luchillo17** [comment]
Have you had to comment this on every PR the stale bot flagged?

@huybuidac It looks nice, one corner case I found is if I set a time and then change the date through the calendar, the time get's reset to my current time, rather than the selected time, the other thing would be that the Storybook has no setup for dark mode, I hoped to see the dark variant of it.

**Comment by shadcn** [comment]
This issue has been automatically marked as stale due to one year of inactivity. It will be closed in 7 days unless there’s further input. If you believe this issue is still relevant, please leave a comment or provide updated details. Thank you. (This is an automated message)

**Comment by shadcn** [comment]
This issue has been automatically closed due to one year of inactivity. If you’re still experiencing a similar problem or have additional details to share, please open a new issue following our current issue template. Your updated report helps us investigate and address concerns more efficiently. Thank you for your understanding! (This is an automated message)
